code/game/objects/structures/fluff.dm

```diff
@@ -276,12 +276,24 @@
 	plane = FLOOR_PLANE
 	deconstructible = TRUE
 
+/obj/structure/fluff/tram_rail/lavaland
```

Comment:
does this need to be on a subtype? feels like lava resistance and fire resistance would be perfectly okay to put on the stationside tram rails too